lib/routes/user/home/account_page.dart

```diff
@@ -57,7 +57,7 @@ class _AccountPageState extends State<_AccountPage> {
     _statusSubscription = widget._accountBloc.accountStream.listen((acc) {
       if (acc.paymentRequestInProgress != null && acc.paymentRequestInProgress.isNotEmpty && acc.paymentRequestInProgress != _paymentRequestInProgress) {        
         Scaffold.of(context).showSnackBar(new SnackBar(
-            duration: new Duration(seconds: 5), content: new Text("Processing Payment...")));
+            duration: new Duration(seconds: 60), content: new Text("Processing Payment...")));
```

Comment:
I guess we want to show the snackbar until and event of either success or failed payment arrived.
I think infinite duration will better fit here.

Reply:
I agree but I couldn't find a way specify an infinite Duration. If parameter left unset it defaults to 4 seconds. Any ideas?

Reply:
Maybe this can work?
new Duration(seconds: Int32.MAX_VALUE.toInt())